x/data/spec/04_events.md

```diff
@@ -1,3 +1,5 @@
 # Events
 
-The `data` module emits events defined in [the Protobuf reference](./protobuf.html#regen-data-v1alpha2-events-proto).
+The data module emits events defined in the Protobuf definitions:
+
+- [regen.data.v1](https://buf.build/regen/regen-ledger/docs/main/regen.data.v1)
```

Comment:
```suggestion
- [regen.data.v1](https://buf.build/regen/regen-ledger/docs/main/regen.data.v1#regen.data.v1.EventAnchorData)
```

should we link to the first event in the page?

Reply:
Hmmm... I was wondering the same thing since there is no event header but the event might change or a new event might be added before it. Maybe not likely but I think it's ok to leave as is for now.